doc/README.de.md

```diff
@@ -0,0 +1,168 @@
+ 4. Baue selbst die APK.
+ Das ist die beste Methode um neue Features schnellstmöglich zu bekommen, aber auch sehr kompliziert und umständlich.
+ War raten dir daher zu einer der anderen Methoden.
+ 5. Falls du an einem konkreten Feature oder Bugfix interessiert bist, welches/r in einem Pull Request in diesem Repo adressiert wird, kannst du auch die APK aus dem PR direkt beziehen.
+ Lies dir die PR Beschreibung durch für entsprechende Anweisungen. Das Gute an PR-spezifischen APKs ist, dass sie parallel zu der offiziellen App installiert werden.
+ Du musst also nicht fürchten, dass deine Daten verloren gehen oder irgendetwas daneben geht.
+
+Für die meisten Benutzer empfehlen wir die erste Methode.
+APKs, die mit Methode 1 oder 2 installiert wurden, sind untereinander kompatibel (d.h. wenn NewPipe mit Methode 1 installiert wurde, kann es trotzdem mit Methode 2 aktualisiert werden und vice versa) - aber nicht mit Methode 3.
+Das liegt daran, dass Methode 1 und 2 den gleichen Signierschlüssel benutzen (unseren), während Methode 3 einen anderen benutzt (den von F-Droid).
+Eine mit Methode 4 gebaute Debug APK benutzt überhaupt gar keinen Schlüssel.
+Das Signieren von Schlüssel ermöglicht Benutzern, dass bösartige Updates nicht versehentlich installiert werden.
+Bei Methode 5 wird für jede APK ein zufälliger Signierschlüssel von GitHub Actions generiert, d.h. Updates sind unmöglich.
+Bei jeder neuen APK Installation muss ein Backup gemacht werden und die Daten wiederhergestellt werden.
+
+Falls du aus welchem Grund auch immer die Quelle (Methode) ändern willst (z.B. weil irgendetwas in NewPipe nicht mehr funktioniert und F-Droid nicht das neuste Update mit dem Fix hat), empfehlen wir folgende Herangehensweise:
+1. Mache ein Back-up über _Einstellungen > Inhalt > Datenbank exportieren_ damit dein Verlauf, deine Abos und Playlists nicht verloren gehen
+2. Deinstalliere NewPipe
+3. Lade die APK von der neuen Quelle herunter und installiere es
+4. Importiere die Daten aus Schritt 1 über _Einstellungen > Inhalt > Datenbank importieren_
+
+<!-- What is an unsupported app anyway? -->
+<b>Hinweis: Wenn du eine Datenbank in die offizielle App importierst, stelle sicher, dass sie _von_ der offiziellen App exportiert wurde.
+Wenn du eine Datenbank von einer nicht offiziellen APK exportierst, werden Dinge vorraussichtlich kaputt gehen.
+So eine Aktion wird nicht unterstützt und du solltest es nur in Erwägung ziehen, wenn du weißt, was du tust.</b>
+
+
+## Beitrag
+Egal ob du neue Ideen, Übersetzungen, Designvorschläge, kleine Code-Bereinigungen, oder sogar große Code-Verbesserungen hast, jegliche Unterstützung ist immer gern gesehen.
+Die App wird mit _jedem_ Beitrag besser und besser - egal wie groß oder klein er sein mag!
+Wenn du dich einbringen willst, sehe dir die [Beitragshinweise](.github/CONTRIBUTING.md) an.
```

Comment:
I rewrote this to avoid double _groß_

```suggestion
Egal ob du neue Ideen, Übersetzungen, Designvorschläge, kleine Code-Bereinigungen, oder sogar große Code-Verbesserungen hast, jegliche Unterstützung ist immer gern gesehen.
Die App wird mit _jedem_ Beitrag besser und besser - egal wie viel Arbeit in ihn gesteckt wird!
Wenn du dich einbringen willst, sehe dir die [Beitragshinweise](.github/CONTRIBUTING.md) an.
```